src/styles/moyo_header_styles/moyo_header.scss

```diff
@@ -0,0 +1,77 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: $white-accent-color;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+
+  &__image {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+
+  &__item {
+    margin-left: 20px;
```

Comment:
Consider using the `:not(:first-child)` pseudo-class to apply the left margin to all items except the first one, instead of setting the margin to all and then resetting it for the first child. This can make the code a bit cleaner.